### DISCUSSION TITLE
Broadcast Whatsapp menggunakan kirimwa.id dengan framwork VUEJS

### DISCUSSION THREAD

**Discussion by akbarism** [kirimwa-id/forum#56] [Pamerkan]
## Whatsapp broadcaster

<img src="https://upload.wikimedia.org/wikipedia/commons/thumb/9/95/Vue.js_Logo_2.svg/2367px-Vue.js_Logo_2.svg.png" height="100px" />
**Dibuat menggunakan VueJs**

## Fitur
- Generate Qr Code auto redirect ke halaman dashboard
- Mengirim pesan sekaligus banyak
- Input nomor bisa export excel atau input manual

## Screenshoot
<div>
<img src="https://user-images.githubusercontent.com/59020048/136905114-d9acb3c8-180d-4d4a-b0ab-fe5a6c6be108.png" height="300px" />

<img src="https://user-images.githubusercontent.com/59020048/136902803-d8aeab7a-3904-42b2-bcb4-e3e30488b6b8.png" height="300px" />

<img src="https://user-images.githubusercontent.com/59020048/136903019-e0b3fb6e-5579-4b8a-987f-907d01ad912d.png" height="300px" />

<img src="https://user-images.githubusercontent.com/59020048/136903061-72897645-08e5-4397-b1e7-4163c8abd685.png" height="300px" />

<img src="https://user-images.githubusercontent.com/59020048/136903099-6fac1ed8-45a1-444a-b844-71b8459926b7.png" height="300px" />

<img src="https://user-images.githubusercontent.com/59020048/136905411-f2cac5e9-603f-48ae-b6bd-fe80461b21cd.png" height="300px" />
</div>

## -
Terimakasihmas [Rio](https://rioastamal.net/), APInya sangat keren sekali :D
Sukses terus...!!

**Comment by rioastamal** [comment]
😎  👍  Mantab mas @akbarism! Terima kasih telah memposting hasil karya menggunakan API KirimWA.id ditunggu update-update berikutnya karena Fitur-fitur baru API KirimWA.id yang segera meluncur diversi berikutnya seperti Scheduled Message, Batch Messages, Webhook Status Perangkat.

**Comment by aminubilhuda** [comment]
Tutorial Cara Makeknya Bang.

**Reply by akbarism** [reply]
sayangnya API kirimwa.id masih kena CORS kalo di production, jadi saya pake buat temen2 kantor aja

**Reply by aminubilhuda** [reply]
waduh bisa ajarin cara instalnya bang?

**Reply by akbarism** [reply]
ada di git saya sih, porjectnya public, di clone aja, token-nya tinggal minta ke mas rio

[repository](https://github.com/akbarism/whatsapp-broadcaster)

**Reply by kafaa-dev** [reply]
> sayangnya API kirimwa.id masih kena CORS kalo di production, jadi saya pake buat temen2 kantor aja

bisa bikin halaman sendiri di backend, terus curl ke Api kirimwa.id, lalu web akan ajax ke backend, biar nggak ketahuan juga tokennya 😉

**Reply by akbarism** [reply]
> > sayangnya API kirimwa.id masih kena CORS kalo di production, jadi saya pake buat temen2 kantor aja
> 
> bisa bikin halaman sendiri di backend, terus curl ke Api kirimwa.id, lalu web akan xhr ke backend, biar nggak ketahuan juga tokennya 😉

eewkwekwe, saya coba dulu ya om

**Reply by akbarism** [reply]
wah bisa, dari kemarin error karena servernya

**Comment by mdestafadilah** [comment]
maaf. mohon infonya ya
https://github.com/akbarism/whatsapp-broadcaster/issues/1
